CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 All changes to the project will be documented in this file.
 
+## [1.32.0] - _Not Yet Released_
+* Added support for code actions in `.cake` files. ([#1205](https://github.com/OmniSharp/omnisharp-roslyn/issues/1205))
```

Comment:
I won the merge race. This will probably have a conflict.

Reply:
Good that I caught my eyes on the `/codestructure` PR. We need to create a Cake endpoint for that too.. Created issue here #1214